can/interface.py

```diff
@@ -27,21 +23,12 @@ def __new__(cls, other, channel=None, *args, **kwargs):
             can.rc['interface'] = kwargs['bustype']
```

Comment:
It doesn't have to be read only (one might still want to change it programmatically), just that creating a bus different from the default should not modify the default.